idaes/models_extra/gas_solid_contactors/unit_models/fixed_bed_1D.py

```diff
@@ -216,32 +228,18 @@ class FixedBed1DData(UnitModelBlockData):
         "pressure_drop_type",
         ConfigValue(
             default="ergun_correlation",
-            domain=In(["simple_correlation", "ergun_correlation"]),
+            domain=In(["ergun_correlation", "simple_correlation"]),
```

Comment:
Not needed now, but for the future you should consider using Python `Enums` for this - they have the benefit of auto-completion in IDEs and documentation as a starting point.